my-react-app/src/components/HomePage.jsx

```diff
@@ -7,91 +7,106 @@ import image1 from '../assets/images/Polygon1.svg';
 import image2 from '../assets/images/Rectangle46.svg';
 import { connect } from 'react-redux';
 import { searchAsync } from '../actions/tutorAction';
+import background_image from '../assets/images/Background_Image.png';
 
 const StyledHomePage = styled.div`
-  background-color: #ffffff;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: 100vh;
-
-  & .div {
-    background-color: #ffffff;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    height: 100vh;   /* Adjusted from a fixed height */
-    width: 100vw;    /* Adjusted from a fixed width */
-    max-width: 1280px;  /* If you still want to limit it to 1280px at larger screens */
-  }
+  background: url(${background_image}) no-repeat center center;
+  background-size: cover;
+  position: relative;
+  width: 100%;
+  height: 500px;
+`;
 
-  & .text-wrapper-8 {
-    color: #000000;
-    font-family: "Inter-Regular", Helvetica;
-    font-size: 1.25rem;   /* Adjusted font size */
-    font-weight: 400;
-    letter-spacing: 0;
-    line-height: normal;
-    white-space: nowrap;
-  }
+const TextWrapper = styled.p`
+  text-align: center;
+  margin-top: 2rem;
+  font-weight: bold;
+  font-family: 'Open Sans', sans-serif;
+`;
 
-  & .rectangle-5 {
-    background-color: #ffffff;
-    border: 1px solid #000000;
-    border-radius: 1rem;  /* Adjusted border radius */
-  }
+const TextWrapper1 = styled.div`
+  text-align: center;
+  margin: 2rem auto;
+  width: 80%;
+  line-height: 1.5;
+  font-weight: bold;
+  font-family: 'Open Sans', sans-serif;
+`;
 
-  & .text-wrapper-12 {
-    color: #000000;
-    font-family: "Inter-Regular", Helvetica;
-    font-size: 2rem;  /* Adjusted font size */
-    font-weight: 400;
-    text-align: center;
-    width: 80%;  /* Adjusted width */
-    margin-top: 2rem;  /* Adjusted margin */
-  }
+const TutorProfilesWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
 `;
 
+const TutorProfile = styled.div`
+  margin-top: 200px; /* Adjust the margin as needed */
+  cursor: pointer;
+  text-align: center;
+`;
 
+const TutorImage = styled.img`
+max-width: 400px; /* Increase this value to make the images larger */
+max-height: 200px; /* Increase this value to make the images larger */
+`;
 
 export const HomePage = ({ tutors_data, tutors_loading, tutors_error }) => {
   const [query, setQuery] = useState('');
   const results = useSelector(state => state.results);
   const dispatch = useDispatch();
-  
-  const handleSearch = (query) => {
+
+  const handleSearch = query => {
     dispatch(searchAsync(query));
   };
 
+  const tutorProfiles = [
```

Comment:
I'd move this to the constants folder just create a file called tutorProfiles.tsx and move this there, then just import it. I'd rather keep this file clean.

Reply:
pls check the new code